.github/workflows/pr-check.yml

```
@@ -0,0 +1,51 @@
+name: PR Check
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened] # 모든 브랜치에 대한 PR 체크
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)